mappings/net/minecraft/util/math/Vec3i.mapping

```diff
@@ -44,7 +45,10 @@ CLASS net/minecraft/class_2382 net/minecraft/util/math/Vec3i
 		ARG 1 direction
 		ARG 2 distance
 	METHOD method_23227 down (I)Lnet/minecraft/class_2382;
+		ARG 1 distance
 	METHOD method_23228 down ()Lnet/minecraft/class_2382;
 	METHOD method_23854 toShortString ()Ljava/lang/String;
+	METHOD method_30558 getComponentAlongAxis (Lnet/minecraft/class_2350$class_2351;)I
```

Comment:
name too obtuse. maybe `getLength` or `getComponent`

Reply:
I named it `getComponentAlongAxis` for consistency with `Vec3d.getComponentAlongAxis`.
 https://github.com/FabricMC/yarn/blob/606ee3910422250a2770b349e5408a77c87d051b/mappings/net/minecraft/util/math/Vec3d.mapping#L56

Is it redundant?

Reply:
It's probably best to go with consistency for now